.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+minimum_pre_commit_version: "2.9.0"
+ci:
+  autofix_prs: false
+exclude: ^src/vendor/
+repos:
+  - repo: https://github.com/Zac-HD/shed
+    rev: 0.9.5
+    hooks:
+      - id: shed
+        # TODO: bump to --py310-plus when Talon moves to Python 3.10.
+        args: [--refactor, --py39-plus]
+        types_or: [python, markdown, rst]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-case-conflict
+      - id: check-executables-have-shebangs
+      - id: check-merge-conflict
+      - id: check-shebang-scripts-are-executable
+      - id: check-symlinks
```

Comment:
```suggestion
      - id: check-symlinks
      - id: check-yaml
```

Reply:
I'm pretty sure prettier will fail if it finds bad yaml, so that seems redundant to me. I can verify that tho

Reply:
Ok confirmed that prettier reports invalid yaml, and the output is prettier than check-yaml